tools/exports.js

```diff
@@ -17,3 +17,6 @@ exports["string_template"] = string_template;
 exports["tokenizer"] = tokenizer;
 exports["is_identifier"] = is_identifier;
 exports["SymbolDef"] = SymbolDef;
+exports["minify"] = minify;
+exports["readNameCache"] = readNameCache;
+exports["writeNameCache"] = writeNameCache;
```

Comment:
Let's not export `readNameCache` & `writeNameCache`, as we never do before anyway.